tttx/calibration/R3BTttxCal2Hit.h

```diff
@@ -55,20 +59,105 @@ class R3BTttxCal2Hit : public FairTask
 
     inline void SetOnline(Bool_t option) { fOnline = option; }
 
+  public:
+    class HitDetector
+    {
+      public:
+        HitDetector() = default;
+        HitDetector(R3BTttxCalData* hit);
+        ~HitDetector();
+        void AddHit(R3BTttxCalData* hit);
+
+        [[nodiscard]] inline const auto& GetX() { return fX; };
+        [[nodiscard]] inline const auto& GetE() { return fE; };
+        [[nodiscard]] inline const auto& GetT() { return fT; };
+        [[nodiscard]] inline const auto& GetDetID() { return fDetID; };
+        [[nodiscard]] inline const auto& IsTrefIn() { return fTrefIncluded; };
+
+      private:
+        double StripID2X(int ID)
+        { // 1-oriented
+            return (static_cast<double>(ID) - static_cast<double>(NStrips) / 2. - 0.5) * WStrip;
+        }
+
+        double fX{ NAN };
+        double fE{ NAN };
+        double fE2{ NAN }; // Squared energy used in AddHit()
+        double fT{ NAN };
+        int fDetID{ -1 };
+        bool fTrefIncluded{ false };
+    };
+
+    class HitsDetector
+    {
+      public:
+        HitsDetector() = default;
+
+        void Init();
+        void NewCluster(R3BTttxCalData* hit);
+        void AddHitToLast(R3BTttxCalData* hit);
+        auto GetMult() { return fHits.size(); };
+        auto& GetHit(int mult) { return fHits.at(mult); }
+        /*
+        [[nodiscard]] inline const auto& GetMult() { return fHits.size(); };
+        auto GetX(int mult) { return fHits.at(mult).fX; };
+        auto GetE(int mult) { return fHits.at(mult).fE; };
+        auto GetT(int mult) { return fHits.at(mult).fT; };
+        */
+      private:
+        std::vector<HitDetector*> fHits;
+        int fDetID;
+    };
+
+    [[nodiscard]] inline const bool& GetRequireTref() { return fRequireTref; }
+    [[nodiscard]] inline const bool& GetStoreOnlyTwoDetHitEvent() { return fStoreOnlyTwoDetHitEvent; }
+    [[nodiscard]] inline const double& GetFixedBeta() { return ffixedbeta; }
+    [[nodiscard]] inline const double& GetTimeWindow() { return ftimewindow; }
+    [[nodiscard]] inline const uint& GetNumNeighbours() { return fneighbours; }
+    void SetRequireTref(bool val) { fRequireTref = val; }
+    void SetStoreOnlyTwoDetHitEvent(bool val) { fStoreOnlyTwoDetHitEvent = val; }
+    void SetFixedBeta(double val) { ffixedbeta = val; }
+    void SetTimeWindow(double val) { ftimewindow = val; }
+    void SetNumNeighbours(uint val) { fneighbours = val; }
+
+    // Method to access from external process for the event-by-event reconstruction.
+    void RecalculateZ(double fbeta = -1.);
```

Comment:
This method is commented in the .cxx

Reply:
Please, comment it to avoid warnings!

Reply:
Sure. I force-pushed with the line commented.